addons/html_editor/static/src/main/font/font_selector.xml

```diff
@@ -6,7 +6,9 @@
             </button>
             <t t-set-slot="content">
                 <t t-foreach="items" t-as="item" t-key="item_index">
-                    <DropdownItem class="'user-select-none'" onSelected="() => this.onSelected(item)">
+                    <DropdownItem onSelected="() => this.onSelected(item)"
+                        t-on-pointerdown.prevent="() => {}"
+                    >
```

Comment:
Does it need to be on three lines? It hinders the syntax coloring of GitHub. If it's the norm to write that kind of stuff on multiple lines then I'm not necessarily against it just for the syntax coloring, but I don't think it's the norm, right?

Reply:
You are right, I didn't pay much attention to that, I fixed it